BitwardenShared/UI/Auth/TwoFactorNotice/EmailAccess/EmailAccessView.swift

```diff
@@ -0,0 +1,100 @@
+import SwiftUI
+import SwiftUIIntrospect
+
+// MARK: - EmailAccessView
+
+/// A view that alerts the user to the new policy of sending emails to confirm new devices.
+///
+struct EmailAccessView: View {
+    // MARK: Properties
+
+    /// An environment variable for getting the vertical size class of the view.
+    @Environment(\.verticalSizeClass) var verticalSizeClass
+
+    /// The `Store` for this view.
+    @ObservedObject public var store: Store<EmailAccessState, EmailAccessAction, EmailAccessEffect>
+
+    var body: some View {
+        VStack(spacing: 12) {
+            DynamicImageTextStackView(minHeight: 0) {
+                Asset.Images.Illustrations.businessWarning.swiftUIImage
+                    .resizable()
+                    .frame(
+                        width: verticalSizeClass == .regular ? 152 : 124,
+                        height: verticalSizeClass == .regular ? 152 : 124
+                    )
+                    .accessibilityHidden(true)
+            } textContent: {
+                VStack(spacing: 16) {
+                    Text(Localizations.importantNotice)
+                        .styleGuide(.title, weight: .bold)
+
+                    Text(Localizations.bitwardenWillSendACodeToYourAccountEmail)
+                        .styleGuide(.title3)
+                }
+                .padding(.horizontal, 12)
+            }
+
+            toggleCard
+                .padding(.horizontal, 12)
+
+            VStack(spacing: 12) {
+                AsyncButton(Localizations.continue) {
+                    await store.perform(.continueTapped)
+                }
+                .buttonStyle(.primary())
+            }
+            .dynamicTypeSize(...DynamicTypeSize.xxxLarge)
+            .padding(.horizontal, 16)
```

Comment:
🤔 I think this might be adding double horizontal padding, since the `scrollView()` modifier also adds horizontal padding. `scrollView()` also adds the background color.